### PR TITLE
implement todo list with react-redux

### PR DESCRIPTION
**Pull request by AlintaJP**

### CODE REVIEW

File: to-do-list/src/todo/components/TodoListItem/TodoListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  };
+
+  return (
+    <li className="item">
+      <span>{text}</span>
+      <RemoveButton
+        type="button"
+        onClick={deleteHandler}
+        className="remove-button"
+        aria-label="todo-remove-button"
```
Comment (siarheikarabko) [inline to-do-list/src/todo/components/TodoListItem/TodoListItem.jsx:18]: Axtually aria-labels are pronounced by screen readers to an user and should be just a plain readable text, smth like - `Remove ${text} todo`.

File: to-do-list/src/todo/components/TodoContainer/TodoContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  const override = css`
+    position: relative;
+    left: 48.5%;
+  `;
+
+  return (
+    <>
+      <TodoForm onAddTodo={addTodoHandler} />
+      <TodoList todos={todos} onDeleteTodo={deleteTodoHandler} />
```
Comment (siarheikarabko) [inline to-do-list/src/todo/components/TodoContainer/TodoContainer.jsx:44]: Just a reminder to discuss a memoization and perfomance.

File: to-do-list/src/todo/components/TodoListItem/TodoListItem.jsx (hunk 1 above)
Comment (AlintaJP) [reply to-do-list/src/todo/components/TodoListItem/TodoListItem.jsx:18]: Fixed

File: to-do-list/src/todo/components/TodoContainer/TodoContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      {status === todoStatus.LOADING ? (
+        <ClipLoader size={50} color="#9c36b5" css={override} />
+      ) : (
+        <Button onClick={addAsyncTodoHandler} classes="async-todo-button">
```
[the remark this answers is not in the document]
Comment (AlintaJP) [reply to-do-list/src/todo/components/TodoContainer/TodoContainer.jsx]: Fixed

**Pull request merged**